### DISCUSSION TITLE
Do you have any plans to change the license to a less permissive one or sell jodit?

### DISCUSSION THREAD

**Discussion by DerKnerd** [xdan/jodit#1133] [General]
Hey @xdan,

this might sound like an odd question, but please bear with me.

I am currently developing a CMS for a friend of mine, my girlfriend and me, it is licensed under the MIT license. Usually all projects I do, are licensed that way, since it is one of, if not the best, OSS license available for hobby projects. For years my go to RTE and WYSIWYG editor was tinyMCE, but just beginning this year they changed the license to GPLv2. Back in 2018 CKEditor did the same. This sparked a proper outrage, I linked the relevant Github issues and discussions below.

Now I am in the process to replace the aging tinyMCE 6 with a new editor since tinyMCE 7 is licensed as GPLv2. My favorite of the few I have found is jodit. Before I switch I would like to know, are there any plans to ditch the MIT license and go back to GPLv2 only?

Thanks for the great work and I hope you keep MIT,
DerKnerd

Here are the promised issues and discussions:

- https://github.com/ckeditor/ckeditor5/issues/991
- https://github.com/tinymce/tinymce/issues/9453
- https://github.com/ckeditor/ckeditor5/issues/14314
- https://github.com/tinymce/tinymce/discussions/9496

**Comment by xdan** [comment]
Hi, no, there are no plans to switch to GPL2 and are not expected

**Reply by DerKnerd** [reply]
Awesome, then you have another (hopefully) happy user :slightly_smiling_face:
